.github/workflows/amplify-deploy.yml

```diff
@@ -106,7 +106,7 @@ jobs:
           aws amplify start-deployment \
             --app-id ${{ env.AMPLIFY_ID }} \
             --branch-name ${{ env.BRANCH_NAME }} \
-            --source-url https://${{ env.BUCKET_NAME }}.s3.${{ env.AWS_REGION }}.amazonaws.com/${{ env.ZIP_NAME }}
+            --source-url s3://${{ env.BUCKET_NAME }}/${{ env.ZIP_NAME }}
```

Comment:
This is much cleaner! Does this fix any particular issues? And do you think this was always possible?

Reply:
I think this was always possible but the problem it solves was the deployments failing on Pie Docs. We'll probably look to change this in other places using this same deployment format.